arm-web/2016-06-01/swagger/connectionGateways.json

```diff
@@ -351,6 +372,38 @@
       "x-ms-azure-resource": true,
       "description": "The base resource type."
     },
+    "UntrackedResource": {
+      "required": [ "location" ],
```

Comment:
As far as I understand it, an "Untracked Resource" should not have a "location" set as required. Also, ARM has been calling this type of resources, "Proxy Resource"